Owncloud iOs Client/Tabs/RecentTab/RecentViewController.m

```diff
@@ -1019,6 +1021,9 @@ - (void) overWriteFile {
     
     FileDto *file = [UploadUtils getFileDtoByUploadOffline:self.selectedUploadToResolveTheConflict];
     
+    ManageThumbnails *manageThumbnails = [ManageThumbnails sharedManager];
```

Comment:
We should use directly the singleton not getting the singleton in a local instance. Just to use the same style on all the app (like the other singletons) :wink: